### PR TITLE
python3 - update setuptools version

### PR DESCRIPTION
**Pull request by adi88d**
## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Content Pull Request
Related PR: link to the PR at demisto/content

## Related Issues
Related: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

### CODE REVIEW

File: docker/python3/Pipfile [hunk 1]
```diff
@@ -15,5 +15,6 @@ funcy = "*"
 tldextract = "*"
 more-itertools = "*"
 defusedxml = "*"
+setuptools="70.3.0"
```
Comment (tkatzir) [inline docker/python3/Pipfile:18]: Still needed?

**Review by tkatzir — COMMENTED** [review]
Great!
One comment.

File: docker/python3/Pipfile (hunk 1 above)
Comment (adi88d) [reply docker/python3/Pipfile]: no need
I revert this file

File: docker/python3/Dockerfile [hunk 2]
```diff
@@ -10,7 +10,8 @@ COPY localtime /etc/localtime
 
 RUN apk --update add --no-cache --virtual .build-dependencies python3-dev build-base wget \
   && pip install --no-cache-dir -r requirements.txt \
-  && apk del .build-dependencies
+  && apk del .build-dependencies \
+  && pip install --upgrade setuptools
```
Comment (tkatzir) [inline docker/python3/Dockerfile:14]: Maybe?
```suggestion
  && pip install --upgrade setuptools>=70.3.0
```

**Review by tkatzir — APPROVED** [review]
Great! Approved.

**Pull request merged**